systemtest/tmt/plans/main.fmf

```diff
@@ -129,6 +127,16 @@ prepare:
       make docker_build
       make docker_tag
       make docker_push
+      
+      # Get latest Kafka version from kafka-versions.yaml      
+      export KAFKA_VERSION=$(cat kafka-versions.yaml | yq eval '.[] | select(.default) | .version' -)
```

Comment:
Btw if you don't want to do it here, you can keep it blank and the script for building the connect image will pick it exactly like this.